client/src/components/table/HomepageRecords.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { FC, useEffect, useMemo } from 'react';
+import { FullPopulatedPitch } from 'ssw-common';
+
+import { buildColumn } from '..';
+import { useAuth } from '../../contexts';
+import { findPendingContributor } from '../../utils/helpers';
+
+import {
+  titleColumn,
+  descriptionColumn,
+  associatedInterestsColumn,
+  deadlineColumn,
+  pitchStatusCol,
+  dateSubmittedCol,
+  associatedTeamsColumn,
+  requestedTeamsColumn,
+  claimStatusColumn,
+  publishDateColumn,
+} from './columns';
+import { PaginatedTable } from './dynamic/PaginatedTable';
+import { DynamicColumn } from './dynamic/types';
+
+const memberCols: DynamicColumn<FullPopulatedPitch>[] = [
```

Comment:
For some reason there's a type error if I don't. I think I have to because FullPopulatedPitch extends BasePopulatedPitch or something, and they get interpreted as the super class by default.